README.md

```diff
@@ -11,16 +11,16 @@ What is Bitcoin?
 Bitcoin is an experimental new digital currency that enables instant payments to
 anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
 with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
+out collectively by the network. Bitcoin Classic is the name of open source
 software which enables the use of this currency.
 
 For more information, as well as an immediately useable, binary version of
-the Bitcoin Core software, see https://www.bitcoin.org/en/download.
+the Bitcoin Classic software, see https://www.bitcoin.org/en/download.
```

Comment:
Should I update the download links to https://bitcoinclassic.com/ or do we already have a placeholder download page?